documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generating-necessary-inputs.rst

```diff
@@ -0,0 +1,84 @@
+
+
+To use ``sct_compute_compression``, we need the 3 following input files:
+
+
+1. Spinal cord segmentation
+2. Vertebral labels
+3. Compression labels (``t2_compressed_label-compression.nii.gz``)
+
+You can also get these files by downloading :sct_tutorial_data:`data_normalizing-morphometrics-compression.zip`.
+
+1. Spinal cord segmentation
+----------------------------
+.. code:: sh
+
+   sct_deepseg_sc -i t2_compressed.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+
+:Output files/folders:
+   - ``t2_compressed_seg.nii.gz`` : Spinal cord segmentation.
+
+
+2. Label the spinal cord vertebral levels
+------------------------------------------
+.. code:: sh
+
+   sct_label_vertebrae -i t2_compressed.nii.gz -s t2_compressed_seg.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : The input image file.
+   - ``-s`` : Spinal cord segmentation.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+:Output files/folders:
+   - ``t2_compressed_seg_labeled.nii.gz`` : Spinal cord vertebral labels.
+
+3. Generate spinal cord compression labels
+-------------------------------------------
+
+1. Open the image in FSLeyes. If the image does not have sufficient axial and sagittal resolution, open two individual images (axial image and sagittal image) in FSLeyes.
+
+.. code:: sh
+
+   fsleyes t2_compressed.nii.gz &
+
+2. Look at the clinical data provided with compression information to know at which level the compression(s) are located.
+3. Locate the compression in the sagittal view.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/localizing_compression_sag.png
+   :align: center
+
+4. In the axial view, toggle to the maximum compressed slice around the compression.
+5. Click on alt+E (Linux/Windows) or option+E (macOS) to open edit mode. Select size 1 and click on the pencil.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/edit_mode.png
+   :align: center
+
+6. Create an empty mask alt+N (Linux/Windows) or option+N (macOS).
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/create_mask.png
+   :align: center
+
+7. Place the label at the center of the spinal cord of the axial image.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/labeling_compression.png
+   :align: center
+
+8. Repeat steps 3 to 7 for the number of compressions.
```

Comment:
This step was a bit confusing to me, because I wasn't sure how many compression labels should be placed.

On the next page, I realized that there are 4 compression sites in total. So, it might be useful to provide the exact points that you used? Something like:

"Note: Our manual labeling team identified 4 compression sites, with labels placed at the following coordinates. Please place labels at those locations if you would like to recreate the sample metric values for this tutorial."

Reply:
>  I wasn't sure how many compression labels should be placed.

This is information which should be provided by clinicians; see [this comment](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4162#discussion_r1301538363).

Reply:
Maybe what we could do is update [Step 2](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4162#discussion_r1301538363) to say "In this example, we were provided with the following clinical observations: [show table with vertebrae]. So, we will be adding 4 compression labels in total."

Then, in Step 8, we can say:

```suggestion
8. Repeat steps 3 to 7 for the remaining three compression sites.
```

That way, it is clear from the outset how many compression labels the user should add during the tutorial.